markdown/guide/what-are-engines.md

```diff
@@ -1,27 +1,43 @@
 ## What Are Engines?
 
```

Comment:
This is a massive rewrite of this section that would be better presented in a separate PR, or multiple PRs. The messaging is very important.

Reply:
moved for next PR